src/hooks/useCheckoutFormStore.ts

```diff
@@ -3,8 +3,10 @@ import { create } from 'zustand';
 const useCheckoutFormStore = create<FormStore>(
   (set) => ({
     formData: {
-      buildTrial: {},
-      createAccount: {},
+      planDetails: {},
+      planDetailsAuthentication: {},
+      accountDetails: {},
+      billingDetails: {},
```

Comment:
Keys could use the enum values from constants to minimize bugs when steps change. Also, we should not store a different state for authenticated vs. unauthenticated Plan Details page.  They both should have all the same fields and underlying state.

```suggestion
      CheckoutStepKey.PlanDetails: {},
      CheckoutStepKey.AccountDetails: {},
      CheckoutStepKey.BillingDetails: {},
```

Reply:
Updated to the enum values as keys, but retained the login and registration fields for now to make the stepper testable.